go/obscuronode/enclave/rollup.go

```diff
@@ -53,12 +53,13 @@ func NewRollup(b *types.Block, parent *Rollup, a common.Address, txs []nodecommo
 	return r
 }
 
-// ProofHeight - return the height of the L1 proof, or -1 - if the block is not known
+// ProofHeight - return the height of the L1 proof, or GenesisHeight-1 - if the block is not known
 // todo - find a better way. This is a workaround to handle rollups created with proofs that haven't propagated yet
-func (r *Rollup) ProofHeight(l1BlockResolver BlockResolver) int {
+func (r *Rollup) ProofHeight(l1BlockResolver BlockResolver) uint64 {
 	v, f := l1BlockResolver.FetchBlock(r.Header.L1Proof)
 	if !f {
-		return -1
+		// return GenesisHeight-1
+		return 0
```

Comment:
Wouldn't this be better as `return obscurocommon.L2GenesisHeight - 1`?

Reply:
yes. that's how it was on the other pr. it got lost.
fixed